LocalPackages/SubscriptionUI/Sources/SubscriptionUI/Preferences/PreferencesSubscriptionModel.swift

```diff
@@ -322,6 +349,38 @@ public final class PreferencesSubscriptionModel: ObservableObject {
         }
     }
 
+    private func currentStorefrontRegion() -> SubscriptionRegion {
+        var region: SubscriptionRegion?
+
+        switch subscriptionManager.currentEnvironment.purchasePlatform {
+        case .appStore:
+            if #available(macOS 12.0, *) {
+                region = subscriptionManager.storePurchaseManager().currentStorefrontRegion
+            }
+        case .stripe:
+            region = .usa
```

Comment:
Is it okay for Stripe to be hardcoded to USA?

Reply:
This is actually correct, region is used to determine feature set to be displayed. For Stripe we do not determine region and operate in "old way" which is only supporting US variant atm.